csharp/private/rules/resgen.bzl

```diff
@@ -0,0 +1,50 @@
+# Label of the template file to use.
+_TEMPLATE = "@d2l_rules_csharp//csharp/private:rules/Template.csproj"
+_TEMPLATE_EMBEDDED_RESOURCE = "        <EmbeddedResource Include=\"%s\" />"
+
+def _csproj_embedded_resource(resx_files):
+    result = ""
+    for src in resx_files:
+        result += _TEMPLATE_EMBEDDED_RESOURCE % (src.basename)

```

Comment:
I'm looking to see if I can figure out a way that uses direct references to the source files as per discussion with @j3parker , as the way I am currently doing this requires copying the `.resx` files into the same directory as the `.csproj`. The issue I am having is that the csproj is looking for the files relative to itself, rather than relative to the workspace.

Experiments with getting the absolute path (or a relative path) from the executing directory of the rule haven't proved very successful.